### PR TITLE
Use new `podinfo` command to gather pod data

### PR DESCRIPTION
**Pull request by jlojosnegros**
From v1.7 `debug-tools` has a new `podinfo` command to help us to gather
info from pods to complement PodResourcesAPI info.
The information gathered is the same that was gotten with the current
`oc` command.

### CODE REVIEW

File: must-gather/collection-scripts/gather_nodes [hunk 1]
```diff
@@ -71,7 +71,7 @@ do
 
     oc exec $pod -n perf-node-gather -- gather_sysinfo snapshot --debug --root=/host --output=- > $NODE_PATH/sysinfo.tgz 2> $NODE_PATH/sysinfo.log
 
-    oc get pods -A --field-selector spec.nodeName=$node,status.phase=Running -o go-template='[{{range $idx, $item := .items}} {{if (ne $idx 0)}},{{end}}{"namespace":"{{.metadata.namespace}}", "name":"{{.metadata.name}}", "nodeName":"{{.spec.nodeName}}", "qosClass": "{{.status.qosClass}}" }{{"\n"}}{{end}}]' > $NODE_PATH/pods_info.json
+    oc exec $pod -n perf-node-gather -- gather_sysinfo podinfo --node-name $node > $NODE_PATH>pods_info.json
```
Comment (fromanirh) [inline must-gather/collection-scripts/gather_nodes:74]: typo: should read `$NODE_PATH/pods_info.json`

Comment (jlojosnegros) [reply must-gather/collection-scripts/gather_nodes]: Big mistake ... Good catch, Thanks!

**Review by fromanirh — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**